tests/e2e/lib/global-teardown.js

```diff
@@ -61,5 +60,5 @@ module.exports = async function ( globalConfig ) {
 	if ( process.env.CI ) {
 		await processSlackLog();
 	}
-	await teardown( globalConfig );
+	// await teardown( globalConfig );
```

Comment:
Minor nitpick: Is this left here for anything in particular or missed in the cleanup?

Reply:
Good catch. I actually forgot about this one, tests not complaining about it... 
This was a call to a jest-puppeteer function and I was not sure what to replace it with, or now, even if it should be replaced at all. 
Maybe @brbrr can help.

Reply:
it coming from `jest-puppeteer` dependency that we using: https://github.com/smooth-code/jest-puppeteer#create-your-own-globalsetup-and-globalteardown

I guess we playwright counterpart does not need that (I'm not sure why tho)